### ISSUE TITLE
श्लि॒षँ॑ - आलिङ्गने, चुरादि , 

### ISSUE THREAD

**Issue by drdhaval2785** [drdhaval2785/SanskritVerb#910]
curAdis don't have aniTtva ever.
Wrong database entry had aniT. So wrong forms.
```
SlekzIzwa-(Sliza!,ASIrliN,ta,curAdi,10.0059)
SlekzIran-(Sliza!,ASIrliN,Ja,curAdi,10.0059)
SlekzIzWAH-(Sliza!,ASIrliN,TAs,curAdi,10.0059)
SlekzIyAsTAm-(Sliza!,ASIrliN,ATAm,curAdi,10.0059)
SlekzIDvam-(Sliza!,ASIrliN,Dvam,curAdi,10.0059)
SlekzIya-(Sliza!,ASIrliN,iw,curAdi,10.0059)
SlekzIvahi-(Sliza!,ASIrliN,vahi,curAdi,10.0059)
SlekzImahi-(Sliza!,ASIrliN,mahiN,curAdi,10.0059)
```

**Comment by drdhaval2785** [comment]
Also systematically analyse any other curAdi having aniTtva.

**Comment by drdhaval2785** [comment]
Only 'ruN' was found out to have this error over and above Sliza!. Corrected in 6b9a4d8491ca905c5364341ecc32a5f29e792e5f
